### PR TITLE
CI build

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

File: tests/test_layers.py [hunk 1]
```diff
@@ -613,7 +613,7 @@ def test_channel_last(self):
         self.assertTrue(self.run_onnx_runtime('channel_last_input', onnx_model, x, expected))
 
     def _test_keras_model(self, model, model_name='onnx_conversion', rtol=1.e-4, atol=1.e-8, img_size=224):
-        img_path = 'data/elephant.jpg'
+        img_path = os.path.join(os.path.dirname(__file__), 'data', 'elephant.jpg')
```
Comment (wenbingl) [inline tests/test_layers.py]: Can you check in this jpeg file as well?

File: .azure-pipelines/win32-conda-CI.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  - task: CondaEnvironment@1
+    inputs:
+      createCustomEnvironment: true
+      environmentName: 'py$(python.version)'
+      packageSpecs: 'python=$(python.version)'
+
+  - script: |
+      conda install -y -c conda-forge protobuf numpy
+      python -m pip install --upgrade pip
+      pip install %COREML_PATH% %ONNX_PATH% cntk==2.5.1
```
Comment (wenbingl) [inline .azure-pipelines/win32-conda-CI.yml]: we don't need cntk and corml stuff here.

File: .azure-pipelines/win32-conda-CI.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      pip install -r requirements-dev.txt
+      pip install %KERAS%
```
Comment (wenbingl) [inline .azure-pipelines/win32-conda-CI.yml]: No onnxruntime for test?

File: tests/test_layers.py (hunk 1 above)
Comment (jiafatom) [reply tests/test_layers.py]: It is already checked-in

File: .azure-pipelines/win32-conda-CI.yml (hunk 2 above)
Comment (jiafatom) [reply .azure-pipelines/win32-conda-CI.yml]: removed

File: .azure-pipelines/win32-conda-CI.yml (hunk 3 above)
Comment (jiafatom) [reply .azure-pipelines/win32-conda-CI.yml]: done

**Pull request merged**